hash/spectrum_cass.xml

```diff
@@ -59561,7 +59612,19 @@ Side B: Once Upon a Lily Pad
 		</part>
 	</software>
 
-
+	<software name="cindy">
+		<description>Cindy Block (Party Version)</description>
+		<year>2025</year>
+		<publisher>YRGB</publisher>
+		<info name="developer" value="Grongy" />
+		<info name="developer" value="Art-top" />
+		<info name="developer" value="n1k-o" />
+		<part name="cass" interface="spectrum_cass">
```

Comment:
Merge those `developer` tags into one.

Reply:
<img width="561" height="578" alt="image" src="https://github.com/user-attachments/assets/58a6c1f8-360a-4cfc-bf5e-8832ca8cd08d" />
Yeah, developers are better be splitted. Same for language

Reply:
In none of the other lists is there repeated information tag (I didn't find it). And, for instance, when there is more than one developer, they are placed under the same tag. 
Regarding the descriptions, for example in Japanese titles, the description in Latin characters is always included on the main description, and in the `alt_title` info tag is placed the description using Katakana characters.
These are my 2 cents... but maybe I'm wrong, someone will clarify it for sure.

Reply:
Let's wait for other comments and maybe create expected directions under docs.mamedev.org

Reply:
for descriptions/titles: https://docs.mamedev.org/techspecs/naming.html

Reply:
My 2c: 
MAME UI is at fault and should newline split on a .csv-like separator (roughly `/` or `,`)

Reply:
We agreed on "description". Do you want me to merge "developers" as well?